song_rater/tests.py

```diff
@@ -29,3 +29,16 @@ def test_add_song_post(self):
     def test_add_song_incomplete_post(self):
         response = self.client.post('/add-song', {'artist': 'Orange'})
         self.assertEqual(response.status_code, 200)
+
+
+class SongListTestCase(TestCase):
+    def setUp(self):
+        song = Song.objects.create(artist='Apple', title='Pie')
+        song.ratings.create(rating=3)
+
+    def test_song_list(self):
+        response = self.client.get('/song-list')
+        self.assertEqual(response.status_code, 200)
```

Comment:
'assertContains' also validates the status code, so this line is not needed.   But it might be that "explicit is better then implicit" [Zen of Python, PEP20] 

https://docs.djangoproject.com/en/3.1/topics/testing/tools/#django.test.SimpleTestCase.assertContains

Reply:
Thank you for bringing this up. I removed the redundant line of code.